examples/DQN/train.py

```diff
@@ -12,160 +12,111 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import argparse
+import os
 import gym
-import paddle.fluid as fluid
 import numpy as np
-import os
+from tqdm import tqdm
+from parl.utils import tensorboard
+from tensorboardX import SummaryWriter
+
 import parl
-from atari_agent import AtariAgent
-from atari_model import AtariModel
-from datetime import datetime
+from model import Model
```

Comment:
maybe it will be better to rename model to cartpole_model, and rename Model to CartpoleModel.